concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-163	163	dol	mad	MAD	1899	7	6	3.25	4.47368421052632	3.7	4.05	5.1	3	1	7983	26	1059	2594	0.187017491186001	0.000114205534948
+Verheyen-2019-1000-164	164	dom	ignorant	BE IGNORANT	2651	5.85	2.95	3.95	2.6	5.55	5.15	2.35	3	1	7030	19	2096	5240	12.6150889861829	0.001863995737802
+Verheyen-2019-1000-165	165	dominant	dominant			10.75	5.5	4.75	6.15	4.3	5.85	3.2	8	3	56622	0	289	113	1.06532303687319	0.000199328440865
+Verheyen-2019-1000-166	166	donker	dismally			4.3	2.7	5.25	4.75	5.15	6.3	2.8	6	2	65866	7	7403	3814	18.5348765867472	0.001867357308328
+Verheyen-2019-1000-167	167	dood	death	DEAD	1625	5.15	1.95	6.2	4.35	5.1	5.75	1.3	4	1	15241	12	6431	40286	26.4497342633645	0.003120052693459
+Verheyen-2019-1000-168	168	doof	deaf	DEAF	996	6.35	3	6.4	3.05	4.6	5.75	2.25	4	1	14912	12	377	817	1.67491707513723	0.00023013253244
+Verheyen-2019-1000-169	169	doordringend	shrill			10.5	4.95	4.2	5.25	2.95	3.65	4.4	12	3	135726	0	233	31	NaN	NaN
+Verheyen-2019-1000-170	170	doorzichtig	white	TRANSPARENT	3754	8	3.15	5.4	3.2	3.7	5.5	4.15	11	3	68215	1	655	108	1.623345848014	0.000301438179449
+Verheyen-2019-1000-171	171	dor	withered			8.9	2.29411764705882	3.85	2.89473684210526	1.75	4.68421052631579	2.6	3	1	12034	20	443	39	0.404431832271844	8.99260792919315E-05
+Verheyen-2019-1000-172	172	dorstig	dry			6.6	4.7	5.2	3.9	3.55	5.45	3.05	7	2	54262	2	98	120	0.031807756557976	3.15004866304515E-06
+Verheyen-2019-1000-173	173	dramatisch	dramatic			9.75	5.8	4.8	5.3	5.05	4.3	2.75	10	3	75903	0	709	379	0.246474203511532	4.28543293839273E-05
+Verheyen-2019-1000-174	174	dreigend	brooding			9.45	5.25	4.65	5.4	3.65	4.8	2	8	2	91214	0	695	12	0.157873970313804	3.42014581080626E-05
+Verheyen-2019-1000-175	175	driehoekig	triangular			5.95	4.1	6.5	4.1	3.75	6.55	3.5	10	3	63125	0	128	8	0.071900847979696	1.55029254449867E-05
+Verheyen-2019-1000-176	176	driftig	irascible			9.7	5.95	3.65	5.21052631578947	2.85	4.25	2.6	7	2	35767	0	566	53	0.159177257547136	6.27471470256577E-05
+Verheyen-2019-1000-177	177	droevig	sad	SAD	699	4.525	2.6	4.85	3.3	5.2	5.05	2	7	2	32947	1	719	453	2.80690963423609	0.00015794174779
+Verheyen-2019-1000-178	178	dromerig	dreamy			7.55	3	3.95	3.2	4.95	4.75	5.05	8	3	83670	1	403	31	0.527045928960507	0.000104986435916
+Verheyen-2019-1000-179	179	dronken	drunk	DRUNK	1463	9.4	5.1	5.9	3.45	5.35	6	3.2	7	2	68872	3	1589	3022	4.16991661046241	0.000512894958503
+Verheyen-2019-1000-180	180	droog	dry	DRY	1398	4.85	2.5	4.95	2.8	5.6	5.65	3.75	5	1	25413	2	2716	1245	3.88047730857054	0.000679967942397
+Verheyen-2019-1000-181	181	druk	busy			7.2	6.05	4.65	5.45	4.8	5.25	3.3	4	1	7577	6	2437	7443	9.06950047714681	0.001328646920027
+Verheyen-2019-1000-182	182	duidelijk	clear	CLEAR (TO SEE)	3710	6.85	4.5	5.4	4.5	6	3.45	5.2	9	3	73207	1	18883	8014	5.80697837898492	0.000595570719178
+Verheyen-2019-1000-183	183	duister	dark	DARK	706	8.2	3.2	4.6	4.95	3.85	5.55	3	7	2	95623	1	1343	649	1.96668606674628	0.000244512764921
```

Comment:
Duister is mapped to DARK, but just like the German equivalent düster, it’s more like gloomy in a light sense, so I am not sure about the mapping. The “standard” adjective for dark would be donker (which is btw not mapped and wrongly translated as “dismally” in the list).

Reply:
GLOOMY 1552 ?

Reply:
Is it the emotional "düster" or when it gets dark in the evening? Our concept of GLOOMY refers to the emotional state of being a bit sad. If it's the second meaning, I would leave it unmapped.

Reply:
As far as I know, it can be both. The getting dark in the evening meaning seems to be the original and more common one, though.

Reply:
Ok thanks! Then I will leave it unmapped.